patch-package
--- a/node_modules/babel-plugin-dotenv/index.js
+++ b/node_modules/babel-plugin-dotenv/index.js
@@ -30,9 +30,9 @@ module.exports = function (data) {
                     }
                     var importedId = specifier.imported.name
                     var localId = specifier.local.name;
-                    if(!(config.hasOwnProperty(importedId))) {
-                      throw path.get('specifiers')[idx].buildCodeFrameError('Try to import dotenv variable "' + importedId + '" which is not defined in any ' + configFile + ' files.')
-                    }
+                    // if(!(config.hasOwnProperty(importedId))) {
+                    //   throw path.get('specifiers')[idx].buildCodeFrameError('Try to import dotenv variable "' + importedId + '" which is not defined in any ' + configFile + ' files.')
+                    // }
 
                     var binding = path.scope.getBinding(localId);
                     binding.referencePaths.forEach(function(refPath){
