patch-package
--- a/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js
+++ b/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js
@@ -39,7 +39,7 @@ function findMatchingSimulator(simulators, simulatorString) {
   var match;
   for (let version in devices) {
     // Making sure the version of the simulator is an iOS or tvOS (Removes Apple Watch, etc)
-    if (!version.startsWith('iOS') && !version.startsWith('tvOS')) {
+    if (!version.includes('iOS') && !version.includes('tvOS')) {
       continue;
     }
     if (simulatorVersion && !version.endsWith(simulatorVersion)) {
@@ -50,7 +50,7 @@ function findMatchingSimulator(simulators, simulatorString) {
       // Skipping non-available simulator
       if (
         simulator.availability !== '(available)' &&
-        simulator.isAvailable !== 'YES'
+        simulator.isAvailable !== true
       ) {
         continue;
       }
