patch-package
--- a/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js
+++ b/node_modules/react-native/local-cli/runIOS/findMatchingSimulator.js
@@ -28,13 +28,13 @@ function findMatchingSimulator(simulators, simulatorName) {
   var match;
   for (let version in devices) {
     // Making sure the version of the simulator is an iOS or tvOS (Removes Apple Watch, etc)
-    if (!version.startsWith('iOS') && !version.startsWith('tvOS')) {
+    if (!version.startsWith('com.apple.CoreSimulator.SimRuntime.iOS') && !version.startsWith('com.apple.CoreSimulator.SimRuntime.tvOS')) {
       continue;
     }
     for (let i in devices[version]) {
       let simulator = devices[version][i];
       // Skipping non-available simulator
-      if (simulator.availability !== '(available)') {
+      if (!simulator.isAvailable) {
         continue;
       }
       let booted = simulator.state === 'Booted';
