patch-package
--- a/node_modules/whatwg-url/lib/URL.js
+++ b/node_modules/whatwg-url/lib/URL.js
@@ -6,11 +6,6 @@ const utils = require("./utils.js");
 const impl = utils.implSymbol;
 
 function URL(url) {
-  if (!new.target) {
-    throw new TypeError(
-      "Failed to construct 'URL'. Please use the 'new' operator; this constructor " + "cannot be called as a function."
-    );
-  }
   if (arguments.length < 1) {
     throw new TypeError(
       "Failed to construct 'URL': 1 " + "argument required, but only " + arguments.length + " present."
